packages/vapor/scss/lib/_mixins.scss

```diff
@@ -23,7 +23,7 @@
     }
 }
 
-@mixin placeholder($text-color: var(--placeholder-text-color), $font-family: var(--main-font)) {
+@mixin placeholder($text-color: var(--default-text-color), $font-family: var(--main-font)) {
```

Comment:
The placeholder should probably keep its color, no?

Reply:
I removed `--placeholder-text-color: var(--grey-80);`since these two are duplicate. 
https://github.com/coveo/react-vapor/pull/1846/files/36467a8f38acdd9b72c136752a10a5c505bd9f92#diff-9cbf75f3c25011738aa63f67ae9d3987443f2923120f40d4e21bd1753412d80dL229